configure.ac

```diff
@@ -259,7 +259,9 @@ AC_ARG_ENABLE(xf86bigfont,    AS_HELP_STRING([--disable-xf86bigfont], [Build XF8
 AC_ARG_ENABLE(dpms,           AS_HELP_STRING([--disable-dpms], [Build DPMS extension (default: enabled)]), [DPMSExtension=$enableval], [DPMSExtension=yes])
 
 dnl kdrive and its subsystems
-AC_ARG_ENABLE(kdrive,         AS_HELP_STRING([--enable-kdrive], [Build kdrive servers (default: yes)]), [KDRIVE=$enableval], [KDRIVE=yes])
+AC_ARG_ENABLE(kdrive,         AS_HELP_STRING([--enable-kdrive],  [Build kdrive servers (default: yes)]), [KDRIVE=$enableval], [KDRIVE=yes])
+AC_ARG_ENABLE(xvesa,          AS_HELP_STRING([--disable-xvesa],  [Build Xvesa server (default: yes)]), [XVESA=$enableval], [XVESA=yes])
```

Comment:
first option should be --enable-xvesa, else the help text doesn't match

Reply:
@rofl0r
I wrote it like that because it's enabled by default, and passing --enable-xvesa is the same as not passing anything.
I saw the same thing happening above too.

Reply:
then the help text should read "Disable build of Xvesa..."

Reply:
btw autoconf automatically generates a --disable-foo switch for every --enable-foo

Reply:
> then the help text should read "Disable build of Xvesa..."

I see.
Should I do the same for --enable-kdrive above, as it's also default-enabled?
By which I mean writing --disable-kdrive and Disable build of kdrive servers?